src/test/java/com/business/entities/Admin_getAdminName_6dc55da125_Test.java

```diff
@@ -0,0 +1,122 @@
+/*
+Test generated by RoostGPT for test java_spring_17Jan using AI Type Open AI and AI Model gpt-4
+
+1. Positive Test Scenario: Check if the function returns the correct admin name when a valid admin name is set.
+
+2. Negative Test Scenario: Check if the function returns null or an empty string when the admin name is not set.
+
+3. Positive Test Scenario: Check if the function returns the updated admin name when the admin name is updated.
+
+4. Negative Test Scenario: Check if the function returns the old admin name even after the admin name is updated.
+
+5. Positive Test Scenario: Check if the function returns the correct admin name when called multiple times.
+
+6. Edge Case Scenario: Check if the function correctly handles and returns special characters if they are part of the admin name.
+
+7. Edge Case Scenario: Check if the function correctly handles and returns long admin names.
+
+8. Edge Case Scenario: Check if the function correctly handles and returns admin names with white spaces.
+
+9. Negative Test Scenario: Check if the function returns an error when trying to fetch an admin name from a non-existent or deleted admin.
+
+10. Positive Test Scenario: Check if the function returns the correct admin name when called simultaneously from different threads (to validate thread-safety).
```

Comment:
PS C:\Users\amank\Spring_Boot_Project> mvn test -Dtest="Admin_getAdminName_6dc55da125_Test"
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------< com.business:SS_Beauty_Products >-------------------
[INFO] Building SS_Beauty_Products 0.0.1-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- resources:3.3.1:resources (default-resources) @ SS_Beauty_Products ---
[INFO] Copying 1 resource from src\main\resources to target\classes
[INFO] Copying 47 resources from src\main\resources to target\classes
[INFO] 
[INFO] --- compiler:3.11.0:compile (default-compile) @ SS_Beauty_Products ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- resources:3.3.1:testResources (default-testResources) @ SS_Beauty_Products ---
[INFO] skip non existing resourceDirectory C:\Users\amank\Spring_Boot_Project\src\test\resources
[INFO]
[INFO] --- compiler:3.11.0:testCompile (default-testCompile) @ SS_Beauty_Products ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- surefire:3.0.0:test (default-test) @ SS_Beauty_Products ---
[INFO] Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.business.entities.Admin_getAdminName_6dc55da125_Test
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.089 s - in com.business.entities.Admin_getAdminName_6dc55da125_Test
[INFO] 
[INFO] Results:
[INFO]
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.982 s
[INFO] Finished at: 2024-01-18T12:06:21+05:30
[INFO] ------------------------------------------------------------------------